src/Nethermind/Nethermind.Evm/TransactionSubstate.cs

```diff
@@ -18,8 +19,25 @@ public class TransactionSubstate
     private const string Revert = "revert";
 
     private const int RevertPrefix = 4;
+    private const int WordSize = EvmPooledMemory.WordSize;
 
     private const string RevertedErrorMessagePrefix = "Reverted ";
```

Comment:
If there was an extra colon in the prefix, i.e. `Reverted: {message}` this would become compatible with Geth. I want to point it out since you're already making a breaking change.

Reply:
There's an ongoing discussion on standarization of error messages: https://github.com/ethereum/execution-apis/issues/463.